test/mocha/10-api.js

```diff
@@ -1,6 +1,204 @@
 /*!
  * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
  */
-describe('api', () => {
-  it('should work');
+import {api, delegateRootZcap} from './helpers.js';
+import {
+  presentations,
+  testExchanges
+} from '../mock.data.js';
+import {
+  shouldError,
+  shouldNotError,
+  shouldHaveVpForStep,
+  shouldHaveVpWithTransactionId,
+  shouldHaveInteractService,
+  shouldBeDelegatedZcap
+} from './assertions.js';
+
+const exchange = '/exchanges/:exchangeId';
+const exchangeInstance = '/exchange-instances/:exchangeInstanceId';
+
+describe('API', () => {
+  describe(exchange, () => {
+    it('should get a Vp from a valid service with only one step.', async () => {
```

Comment:
I don't think we normally put periods/full stops in test descriptions (unless I'm totally misremembering):

```suggestion
    it('should get a VP from a valid service with only one step', async () => {
```